lib/phoenix.ex

```diff
@@ -10,6 +10,8 @@ defmodule Phoenix do
     * [Phoenix](https://hexdocs.pm/phoenix) - the Phoenix web framework
       (these docs)
 
+    * [Phoenix.js](/js) - Phoenix Channels JavaScript client
+
```

Comment:
`[Phoenix.js](js)` will work. That's why we use a flat structure in ExDoc, so we can link using relative paths. :)